src/nvim/os/win_defs.h

```diff
@@ -6,16 +6,22 @@
 #define TEMP_DIR_NAMES {"$TMP", "$TEMP", "$USERPROFILE", ""}
 #define TEMP_FILE_PATH_MAXLEN _MAX_PATH
 
-// Defines needed to fix the build on Windows:
-// - DFLT_DIR
-// - DFLT_BDIR
-// - DFLT_VDIR
-// - EXRC_FILE
-// - VIMRC_FILE
-// - SYNTAX_FNAME
-// - DFLT_HELPFILE
-// - SYS_VIMRC_FILE
-// - SPECIAL_WILDCHAR
+// Windows system-dependent file names
+#ifndef SYS_VIMRC_FILE
+# define SYS_VIMRC_FILE "$VIM\\sysinit.vim"
+#endif
+#ifndef DFLT_HELPFILE
+# define DFLT_HELPFILE  "$VIMRUMTIME\\doc\\help.txt"
+#endif
+#ifndef SYNTAX_FNAME
+# define SYNTAX_FNAME   "$VIMRUNTIME\\syntax\\%s.vim"
```

Comment:
Let's move all of these up to `globals.h` (unless you find somewhere more appropriate) and just concat `PATHSEPSTR` to make the path using the correct separator. E.g.:

```c
#ifndef SYNTAX_FNAME
# define SYNTAX_FNAME   "$VIMRUNTIME" PATHSEPSTR "syntax" PATHSEPSTR "%s.vim"
#endif
```

It seems weird that `PATHSEPSTR` is defined in `ascii.h`, might need to move that, or define a duplicate concept specifically for use in globals.h.

After this we can delete the corresponding lines in unix_defs.h.